tests/test_keras_integration.py

```diff
@@ -533,14 +535,20 @@ def test_push_to_hub_keras_via_http_with_model_kwargs(self):
 
 
 @require_tf
-class HubKerasFunctionalTest(HubKerasSequentialTest):
+class HubKerasFunctionalTest(CommonKerasTest):
     def model_init(self):
         inputs = tf.keras.layers.Input(shape=(2,))
         outputs = tf.keras.layers.Dense(2, activation="relu")(inputs)
         model = tf.keras.models.Model(inputs=inputs, outputs=outputs)
         model.compile(optimizer="adam", loss="mse")
         return model
 
+    def model_fit(self, model):
```

Comment:
What do you say if we take this outside given it just doesn't matter according to how model is built? (instead of making this a class specific method)

Reply:
I thought about that but it's maybe preferable to keep it close to each `model_init` so that they can evolve together. Here the fact that `model_fit` is the same is just "by chance" that both `model_init` takes the same input, right ?

Reply:
I'm totally fine with putting it outside though. Your call !

Reply:
Sure, let's keep it your way.